saleor/discount/migrations/0046_migrate_sales_to_promotions.py

```diff
@@ -1,11 +1,17 @@
 # Generated by Django 3.2.18 on 2023-07-06 09:43
-from django.db import migrations
+from decimal import Decimal
+
+import graphene
 from dataclasses import dataclass
 from typing import Dict, List
 
-import graphene
+from babel.numbers import get_currency_precision
+from django.db import migrations
+from django.apps import apps as registry
 from django.db.models import Exists, OuterRef
+from django.db.models.signals import post_migrate
 
+from .tasks.saleor3_17 import update_discounted_prices_task
```

Comment:
https://docs.saleor.io/docs/3.x/developer/community/zero-downtime-migrations#additional-autodiscovery-with-specified-related-names

additional configuration is needed in this case